### ISSUE TITLE
Blur background and darken background affecting game background

### ISSUE THREAD

**Issue by TheTabbingman** [darklinkpower/Harmony#25]
### Bug Description

The Blur background and darken background in the general appearance settings is affecting the game background aswell as the window background.
In the past it only affected the window's background. And these settings are greyed out until you enable the window background. So I don't think they are supposed to be affecting the game background.

**Comment by darklinkpower** [comment]
Hi, I'm having trouble understanding what the issue is. These settings affect the game background image when enabled:

![image](https://github.com/darklinkpower/Harmony/assets/1389286/bc09ea77-ffa9-42bb-9f4d-177f2b3aa1d6)

Can you elaborate what the issue is with screenshots?

**Comment by darklinkpower** [comment]
> Can you elaborate what the issue is with screenshots?

**Comment by darklinkpower** [comment]
That's not correct, these settings only affect the game window background, which is the blue one. The red one should be unaffected but I just checked and the issue is from the BackgroundChanger extension, which for some reason is blurring its background. I don't know if this is theme related or not since it doesn't do anything to enforce blur but I'll look into it.

**Comment by TheTabbingman** [comment]
Thank you, disabling the BackgroundChanger extension does fix it.

**Comment by darklinkpower** [comment]
I looked into the extension code to see how it worked and I'm afraid that this is something that needs to be fixed in the extension itself, ideally with separate options to enable these settings. Right now it is forcefully copying the settings used by the window background image with no way to separate them.
